task_4/test_downloading.py

```diff
@@ -0,0 +1,24 @@
+from task_4 import df,file_path,get_image
+import pandas as pd
+from pathlib import Path
+import pytest
+import os
+
+# test if a pandas datafram is extracted
+def test_dataframe():
+    assert isinstance(df,pd.DataFrame) == True
+
+# test if file exsist and its a parquet file
+def test_path():
+    assert Path(file_path).is_file() and file_path.endswith('.parquet')
+
+# testing downloading
+@pytest.mark.parametrize("url, i, index",[("https://images.squarespace-cdn.com/content/54efd71fe4b06377d0b2fb17/1525152563529-4XIS44FIBU4N7JCZLEXD/20180429+-+Juanes+-+Toronto+Music+Photography+-+Captive+Camera-5391.jpg?content-type=image%2Fjpeg",13,1),

```

Comment:
use pytest fixtures instead of having raw values here. See the examples in https://docs.pytest.org/en/6.2.x/fixture.html